src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
+  >
+    <div
```

Comment:
Hi, 
1. I feel this might work without the div. Can you please try it?
2. The margin is it coming from the designers? I can see in their designs sometimes is 9, some times 9.333, sometimes 10. Probably this needs to be 8px the same like convert and pay.
3.  Do we need the data-test of the icon to change according to what icon we pass?

Reply:
> 2. The margin is it coming from the designers? I can see in their designs sometimes is 9, some times 9.333, sometimes 10. Probably this needs to be 8px the same like convert and pay.

yes, it was on the figmas. I reduced it to 8.

Reply:
> 3. Do we need the data-test of the icon to change according to what icon we pass?
you are right, i removed the dynamic part

Reply:
> 3. Do we need the data-test of the icon to change according to what icon we pass?
ok, corrected, thanks